pyccel/parser/semantic.py

```diff
@@ -903,6 +903,19 @@ def _handle_function_args(self, arguments, **settings):
         return args
 
     def _handle_function(self, expr, func, args, **settings):
+        """
+        Create a FunctionCall or an instance of a PyccelInternalFunction
+        from the function informations and arguments
```

Comment:
Isn't "information" an uncountable noun in English?

Reply:
Fixed in 47de259